src/pages/home/report/ReportActionCompose.js

```diff
@@ -181,6 +181,11 @@ class ReportActionCompose extends React.Component {
             if (this.shouldFocusInputOnScreenFocus && this.props.isFocused) {
                 this.focus(false);
             }
+
+            // when size of reportActions it means a new chat and we have to focus the input on foucsable devices
+            if (_.size(this.props.reportActions) === 1) {
```

Comment:
Ok. It seems this is not working. It's not focusing at all and it should never work.

You are adding the focus logically in the wrong place.

So, Interest is that focus the composer only when the chat is new and when the user opens the chat screen.

Reply:
can you help me out with the code suggestion please

Reply:
I think you can directly move this to autofocus prop.

Reply:
ok moved to auto focus 
 autoFocus={this.shouldFocusInputOnScreenFocus || _.size(this.props.reportActions) === 1}